build-support/bin/BUILD

```diff
@@ -1,87 +1,21 @@
 # Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
```

Comment:
Most the content is the same here—outside of swapping `sources` for `entry_point`. It's reformatted to be more terse and use Black's formatting style, lmk if you think this is a worse than before.

Reply:
Hmm, applying Black to BUILD files is a really good idea! And a user has asked for something similar IIRC. Would be great to figure out a way for Pants to do so in the future.